packages/pagination/src/add-pagination-body.interceptor.ts

```diff
@@ -94,7 +94,7 @@ export class PaginationBodyInterceptor<T> implements NestInterceptor<DataToPagin
     project?: unknown,
   ): string => {
     // tslint:disable-next-line
-    let url = `${path}?page=${page}&limit=${limit}`;
+    let url = `${path}?page=${page}&${this.perPageName}=${limit}`;
```

Comment:
Why only `perPageName` and not `pageName`?

```suggestion
    let url = `${path}?${this.pageName}=${page}&${this.perPageName}=${limit}`;
```